DependencyInjection/Configuration.php

```diff
@@ -318,6 +319,7 @@ public function getConfigTreeBuilder()
                             ->scalarNode('min_level')->defaultValue('DEBUG')->end() // filter
                             ->scalarNode('max_level')->defaultValue('EMERGENCY')->end() //filter
                             ->scalarNode('buffer_size')->defaultValue(0)->end() // fingers_crossed and buffer
+                            ->scalarNode('flush_on_overflow')->defaultValue(false)->end() // buffer
```

Comment:
I fixed that line. Btw. https://github.com/symfony/MonologBundle/blob/master/DependencyInjection/Configuration.php#L302 :D